browser/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@0x/mesh-browser",
+    "version": "3.0.1-beta",
```

Comment:
My preference would be to set this version to `development` in the development branch, but unfortunately that doesn't really work with Yarn/NPM. Okay with just setting it to the last released version?

Reply:
Yep, that's fine.